exo-go/src/service_helpers/index.go

```diff
@@ -0,0 +1,30 @@
+package serviceHelpers
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/Originate/exosphere/exo-go/src/types"
+	"github.com/Originate/exosphere/exo-go/src/util"
+)
+
+// VerifyServiceDoesNotExist forces the program to exit with status code 1
+// if the service serviceRole already exists in existingServices
+func VerifyServiceDoesNotExist(serviceRole string, existingServices []string) {
+	if util.DoesStringArrayContain(existingServices, serviceRole) {
+		fmt.Printf(`Service %v already exists in this application`, serviceRole)
+		os.Exit(1)
```

Comment:
I was just following the original implementation and feature tests. So if we don't exit, do we just expect the users to do `Ctr-C` after they see the error?